src/components/externalLinks/ExternalLinks.component.tsx

```diff
@@ -214,21 +306,29 @@ export const AppLevelExternalLinks = ({
     }
 
     return (
-        appLevelExternalLinks.length > 0 && (
-            <div
-                data-testid="external-links-wrapper"
-                className="app-level__external-links flex left w-100 dc__border-bottom-n1 bcn-0"
-            >
-                {!isOverviewPage && (
-                    <div className="app-level__external-links-icon icon-dim-20">
-                        <LinkIcon className="external-links-icon icon-dim-20 fc-9" />
+        <>
+            {appLevelExternalLinks.length > 0 && (
+                <div
+                    data-testid="external-links-wrapper"
+                    className="app-level__external-links flex left w-100 dc__border-bottom-n1 bcn-0"
+                >
+                    {!isOverviewPage && (
+                        <div className="app-level__external-links-icon icon-dim-20">
+                            <LinkIcon className="external-links-icon icon-dim-20 fc-9" />
+                        </div>
+                    )}
+                    <div className="flex left flex-wrap dc__gap-8">
+                        {appLevelExternalLinks.map((link, idx) => getExternalLinkChip(link, idx))}
                     </div>
-                )}
-                <div className="flex left flex-wrap">
-                    {appLevelExternalLinks.map((link, idx) => getExternalLinkChip(link, idx))}
                 </div>
-            </div>
-        )
+            )}
+            {expandedExternalLink && (
+                <ExternalLinkIframeModal
+                    selectedExternalLink={expandedExternalLink}
+                    handleCloseModal={handleCloseModal}
+                />
+            )}
+        </>
     )
 }
 
```

Comment:
Suggestion: we use image with fallback in getExternalLinkIcon as well? and if possible give width and height?